main.go

```diff
@@ -325,6 +328,23 @@ func main() { //nolint:funlen,maintidx
 		os.Exit(1)
 	}
 
+	ons, err := cluster.GetOperatorNamespace()
+	if err != nil {
+		setupLog.Error(err, "unable to determine Operator Namespace")
+		os.Exit(1)
+	}
+
+	gc.Instance = gc.New(
+		oc,
+		ons,
+		gc.WithUnremovables(gvk.CustomResourceDefinition, gvk.Lease),
+	)
+
+	err = mgr.Add(gc.Instance)
+	if err != nil {
+		setupLog.Error(err, "unable to register GC service")
```

Comment:
Should we add `os.Exit(1)` for this, given it will return errors later

Reply:
done